ViAn/Project/bookmark.cpp

```diff
@@ -56,7 +56,8 @@ void Bookmark::set_id(ID new_id) {
 }
 
 void Bookmark::reset_root_dir(const std::string &dir){    
-    m_file = dir + Utility::name_from_path(m_file);
+    m_file = dir + "Bookmarks/" + Utility::name_from_path(m_file);
+    m_thumbnail_path = dir + "_thumbnails/" + Utility::name_from_path(m_thumbnail_path);
```

Comment:
Where would be the best place for these constants? Project, img_gen, bookmark, savable or somehting else?

Reply:
Depends on the scope they are used in. If they are used in Project I would place them there since it is basically the base of everything else

Reply:
Thats what i thought too. Right now they are at least used in project, videoproject and image generator so project sound good,